llvm/lib/Target/AMDGPU/VOPInstructions.td

```diff
@@ -111,7 +111,7 @@ class VOP3_Pseudo <string opName, VOPProfile P, list<dag> pattern = [],
 
   bit HasFP8DstByteSel = P.HasFP8DstByteSel;
 
-  let AsmOperands = !if(isVop3OpSel,
+  let AsmOperands = !if(!and(!not(P.IsTrue16), isVop3OpSel),
```

Comment:
I believe the intent is that we can remove AsmVOP3OpSel. When will we be able to do that? Can you make sure there is a comment about that on class getAsmVOP3OpSel ?

Reply:
added a comment in class getAsmVOP3OpSel